src/components/atoms/_drawer.scss

```
@@ -0,0 +1,85 @@
+.tk-drawer-backdrop {
+  &.tk-drawer--relative-to-window {
+    position: fixed;
+  }
+  &.tk-drawer--relative-to-parent {
+    position: absolute;
+  }
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  transition: transform 0.5s ease;
+  background: $--tk-drawer-backdrop-color;
+  z-index: $z-index-drawer-backdrop;
+}
+
+.tk-drawer {
+  &--relative-to-window {
+    position: fixed;
+  }
+  &--relative-to-parent {
+    position: absolute;
+  }
+
+  top: 0;
+  height: -webkit-fill-available;
+  transition: transform 0.5s ease;
+  z-index: $z-index-drawer;
+
+  display: flex;
+  flex-direction: column;
+  @extend .tk-p-2;
+
+  background-color: $--tk-drawer-background-color;
+
+  &--right {
```

Could you group the `&--right` and `&--left`? :)
```suggestion
  &--right {
    right: 0;
    border-left: 1px solid $--tk-drawer-border;
    &.tk-drawer-hidden {
      -webkit-transform: translateX(100%);
      transform: translateX(100%);
    }
  }
```